utils/ruleset-whitelist.csv

```diff
@@ -1710,6 +1710,7 @@ cb7a292994a2fc87f1440d181779f6301b3483c21e5df5c3d7e9ffaaff78770c,1,1,GNOME.xml
 5ddfa3a482674b780d4e0cd81e3ba0264703fe2b42c797a132127a132df0f974,1,1,Gnumeric.org.xml
 25d4ef946d891943dc472eedd8411e05c4ba679047a1b058f9aac1c2611d0cd9,1,1,GoAruna.com.xml
 d71b4a621557b40535fbc3c3d25ac92b0529a901b82e9ae46bcd939451ba9f97,1,1,GO.com.xml
+660db2609d3abb3b65508b3a1be7cbb60fc2a1d82309709e5da013a0b009d278,0,1,GoDaddy.xml
```

Comment:
Please remove this so we can re-test with the fix applied in https://github.com/EFForg/https-everywhere/commit/db8d6495dddb1c954faf2c9f4b65d4320a873e86